lib/compat/experimental/blocks.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Temporary compatibility shims for features present in Gutenberg.
+ *
+ * @package gutenberg
+ */
+
+if ( ! function_exists( 'build_comment_query_vars_from_block' ) ) {
+	/**
+	 * Helper function that constructs a comment query vars array from the passed block properties.
+	 *
+	 * It's used with the Comment Query Loop inner blocks.
+	 *
+	 * @param WP_Block $block Block instance.
+	 *
+	 * @return array Returns the comment query parameters to use with the WP_Comment_Query constructor.
+	 */
+	function build_comment_query_vars_from_block( $block ) {
+		$comment_args = array(
+			'orderby'                   => 'comment_date_gmt',
+			'order'                     => 'ASC',
```

Comment:
Should we create also a toggle for 'ASC' / 'DSC'? Maybe it would be a quick PR to land.

Reply:
Yes, we should add UI controls and the fallback for the site options that already exist when nothing is selected.

Reply:
Added here #37297  I hope it won't interfere too much with this PR.

Reply:
@c4rl0sbr4v0, great idea, thank you for starting new PR 💯